src/components/CardMedia.d.ts

```diff
@@ -1,6 +1,7 @@
-declare const CardMedia: ({ data, cardSize, autoPlay }: {
+declare const CardMedia: ({ data, cardSize, autoPlay, isShowDescription }: {
     data: any;
     cardSize: any;
     autoPlay: any;
+    isShowDescription: any;
```

Comment:
Shall we make it as boolean ? :thinking:

Reply:
Done